index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mobile version</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <a class="logo">My logo</a>
+        <button type="button" class="btn">
+          <img alt="nav" class="menu" src="./images/Union.png" />
+        </button>
+      </nav>
+    </header>
+    <section class="flex-container">
+      <div class="text">
+        <h1>I'm Loic<br />Glad to see<br />you</h1>
+      </div>
```

Comment:
- [ OPTIONAL ] Since `<h1>` is a block level element, what do you think if we remove the wrapper `<div>`? I believe you can do it without the `<div>` but a leaner code.